mpqp/core/instruction/gates/native_gates.py

```diff
@@ -177,6 +237,31 @@ def to_other_language(
                     "export, this feature is coming very soon!"
                 )
             return self.braket_gate(theta)
+        if language == Language.QASM2:
+            from mpqp.qasm.mpqp_to_qasm import float_to_qasm_str
+
+            instruction_str = self.qasm2_gate
+            if isinstance(self, (Rk, CRk)):
```

Comment:
if we have a special case for those gates, should this go directly in the body of these classes ?

Reply:
I agree, but the problem is that you have to redo to_other_language every time, with all the other languages.

Reply:
what about defining it for most cases here, but overloading it for `Rk` et al ?

Reply:
Done: I overloading to_other_language for `Rk` and `CRk`. And i create a task to make the overload function more flexible to avoid rewriting all the functions for the specific case.

Reply:
you can call `super().to_other_language` for the cases in which the overloading is not needed